_docs/manage/features/index.md

```diff
@@ -16,6 +16,8 @@ storage features. No feature labels are present by default.
 | Caching     | `storageos.feature.nocache`     | true / false   | Switches off caching. |
```

Comment:
Do these labels need updating to storageos.com?

Reply:
these are dataplane settings but yeah, eventually once they support it we will need to use storageos.com/ labels

Reply:
I missed that these labels have a different prefix and are not consistent with the old - I wouldn't have approved.

So, do we:
1. revert prefix on new features
1. change prefixes on old features (and provide migration/deprecation notices since there are a lot of docs/tutorials to update)
1. accept the inconsistency

I'm against 3, and suggest we don't publish the feature until we've done either 1 or 2.